metadata.yaml

```diff
@@ -36,6 +36,8 @@ resources:
 provides:
   prometheus-scrape:
     interface: prometheus
+  metrics-endpoint: # For integration with prometheus-k8s
+    interface: prometheus_scrape
```

Comment:
I have changed the relation names according to your suggestion. I had kept the default because [it's recommended to use the same(metrics-endpoint) name](https://github.com/canonical/prometheus-k8s-operator/blob/main/lib/charms/prometheus_k8s/v0/prometheus_scrape.py#L45). At the same time, I also think it makes sense to have clearer names in our case where we have two different but similar relations.

Reply:
I see the value of having the same relation name across charms. Perhaps we should have a team discussion and settle on a name convention for this case where two Prometheus relations co-exist.

Reply:
I think that the `prometheus-legacy-scrape` and `prometheus-k8s-scrape` is good naming for this charm.